src/mtool/ui/src/containers/IbofOsOperations/Subsystem/index.js

```diff
@@ -35,6 +35,7 @@ import { Grid, IconButton, ThemeProvider, Typography, Tooltip, withStyles } from
 import MaterialTable from '@material-table/core';
 import { Add, Check, Clear, FirstPage, LastPage, Search, ChevronRight, ChevronLeft, Delete, Remove, ArrowUpward } from '@material-ui/icons';
 import { connect } from 'react-redux';
+import TablePagination from '@mui/material/TablePagination';
```

Comment:
Remove this import. Instead import it form the Material table. You can change line 35 to

import MaterialTable, {MTablePagination} from '@material-table/core';